<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="C40FF62PC" name="com.rsi.kma.automation.testScripts.Regression.Overview" tests="5" failures="1" timestamp="03 May 2018 16:36:10 GMT" time="889.965" errors="0">
  <testcase name="OverviewEV1" time="311.546" classname="com.rsi.kma.automation.testScripts.Regression.Overview"/>
  <testcase name="OverviewEV2" time="259.280" classname="com.rsi.kma.automation.testScripts.Regression.Overview">
    <failure type="java.lang.AssertionError" message="batteryStatusToBeMatch Not matched">
      <![CDATA[java.lang.AssertionError: batteryStatusToBeMatch Not matchedat org.junit.Assert.fail(Assert.java:88)at com.rsi.kma.automation.modules.OverviewModule.OverviewEV2(OverviewModule.java:287)at com.rsi.kma.automation.testScripts.Regression.Overview.OverviewEV2(Overview.java:33)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:497)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- OverviewEV2 -->
  <testcase name="OverviewConnected" time="167.299" classname="com.rsi.kma.automation.testScripts.Regression.Overview"/>
  <testcase name="OverviewPHEV" time="66.673" classname="com.rsi.kma.automation.testScripts.Regression.Overview"/>
  <testcase name="OverviewKH" time="85.167" classname="com.rsi.kma.automation.testScripts.Regression.Overview"/>
</testsuite> <!-- com.rsi.kma.automation.testScripts.Regression.Overview -->
